InspectorFunctions.py

```diff
@@ -0,0 +1,143 @@
+                        , Result([], "part_of_line_with_invalid_internal_topology_unique_vertices",0)
+                        , Result([], "part_of_invalid_unique_line_unique_vertices", 0)
+                        , Result([], "part_of_lines_but_not_corner_unique_vertices", 0)
+                        , Result([], "part_of_invalid_surfaces_unique_vertices", 0)
+                        ]
+    Wrapper_TopologyTests = Result(TopologyTests, "topology", True)
+    return Wrapper_TopologyTests
+
+def Inspectors():
+
+    BRep_Tests = [Result([TopologyTests("brep")], "BRep", True)]
+    CrossSection_Tests = [Result([TopologyTests("section")], "CrossSection", True)]
+    EdgedCurve2D_Tests = [Result([ColocationTests(), DegenerationTests()], "EdgedCurve2D", True)]
+    EdgedCurve3D_Tests = [Result([ColocationTests(), DegenerationTests()], "EdgedCurve3D", True)]
+    Graph_Tests = [Result([], "Graph", True, True)]
+    HybridSolid3D_Tests = [Result([AdjacencyTests(), ColocationTests(), DegenerationTests(), Result([ManifoldTests("edges"), ManifoldTests("facets"), ManifoldTests("vertices")], "manifold", True)], "HybridSolid3D", True)]
+    PointSet2D_Tests = [Result([ColocationTests()], "PointSet2D", True)]
+    PointSet3D_Tests = [Result([ColocationTests()], "PointSet3D", True)]
+    PolygonalSurface2D_Tests = [Result([AdjacencyTests(), ColocationTests(), DegenerationTests(), Result([ManifoldTests("edges"), ManifoldTests("vertices")], "manifold", True)], "PolygonalSurface2D", True)]
+    PolygonalSurface3D_Tests = [Result([AdjacencyTests(), ColocationTests(), DegenerationTests(), Result([ManifoldTests("edges"), ManifoldTests("vertices")], "manifold", True)], "PolygonalSurface3D", True)]
+    PolyhedralSolid3D_Tests = [Result([AdjacencyTests(), ColocationTests(), DegenerationTests(), Result([ManifoldTests("edges"), ManifoldTests("facets"), ManifoldTests("vertices")], "manifold", True)], "PolyhedralSolid3D", True)]
+    RegularGrid2D_Tests = [Result([], "RegularGrid2D", True, True)]
+    RegularGrid3D_Tests = [Result([], "RegularGrid3D", True, True)]
+    Section_Tests = [Result([TopologyTests("section")], "Section", True)]
+    StructuralModel_Tests = [Result([TopologyTests("brep")], "StructuralModel", True)]
+    TetrahedralSolid3D_Tests = [Result([AdjacencyTests(), ColocationTests(), DegenerationTests(), Result([ManifoldTests("edges"), ManifoldTests("facets"), ManifoldTests("facets"), ManifoldTests("vertices")], "manifold", True)], "TetrahedralSolid3D", True)]
+    TriangulatedSurface2D_Tests = [Result([AdjacencyTests(), ColocationTests(), DegenerationTests(), Result([ManifoldTests("edges"), ManifoldTests("vertices")], "manifold", True)], "TriangulatedSurface2D", True)]
+    TriangulatedSurface3D_Tests = [Result([AdjacencyTests(), ColocationTests(), DegenerationTests(), Result([ManifoldTests("edges"), ManifoldTests("vertices")], "manifold", True)], "TriangulatedSurface3D", True)]
+    VertexSet_Tests = [Result([], "VertexSet", True, True)]
```

Comment:
@MelchiorSchuh Can you tell me if I forgot some tests? I double checked and it seems good to me but you never know...